examples/tv-app/android/java/MediaPlaybackManager.cpp

```diff
@@ -190,23 +310,62 @@ void MediaPlaybackManager::HandleStartOver(CommandResponseHelper<Commands::Playb
 
 bool MediaPlaybackManager::HandleActivateAudioTrack(const chip::CharSpan & trackId, const uint8_t & audioOutputIndex)
 {
-    // Handle Activate Audio Track
-    HandleMediaRequest(MEDIA_PLAYBACK_REQUEST_ACTIVATE_AUDIO_TRACK, 0);
-    return true;
+    return HandleActivateTrack(true, trackId);
+}
+
+bool MediaPlaybackManager::HandleActivateTrack(bool audio, const chip::CharSpan & trackId)
+{
+    std::string id(trackId.data(), trackId.size());
+
+    jint ret       = -1;
+    CHIP_ERROR err = CHIP_NO_ERROR;
+    JNIEnv * env   = JniReferences::GetInstance().GetEnvForCurrentThread();
+
```

Comment:
Could you put the below JniLocalReferenceManager in the beginning of this HandleRecordProgram callback function so that we can automatically release local reference , when it is out of scope.
JNIEnv * env = JniReferences::GetInstance().GetEnvForCurrentThread();
VerifyOrReturn(env != nullptr, ChipLogError(Controller, "Could not get JNIEnv for current thread"));
JniLocalReferenceManager manager(env);

Reply:
fixed - thanks!